brave/brave-core
# Update Rewards connect account modal

Resolves https://github.com/brave/brave-browser/issues/33258



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

Only the UI for the connect wallet has been updated.

- Ensure UI matches [designs](https://www.figma.com/file/eqdxJmEUTSUrTJhYN21Cqp/BAT-on-chain-payout?type=design&node-id=5031-152395&mode=design&t=jy8wL8o2sdDf1GeK-0), with the following exceptions:
  - "Self-custody" section should not be displayed
  - Navigation "dots" should not be displayed
- Ensure that layout is appropriate for narrow windows/screens
- Regression testing
  - Ensure that links and buttons work as expected
